ipam/docs/RangeAPI.md

```diff
@@ -102,12 +102,10 @@ import (
 
 func main() {
     id := "id_example" // string | An application specific resource identity of a resource
-    contiguous := true // bool | Indicates whether the IP addresses should belong to a contiguous block.  Defaults to _false_. (optional) (default to false)
```

Comment:
Why is this being removed?

Reply:
Hmm, Not sure why. Let me try generating the code again

Reply:
Fixed